### PR TITLE
Scrape term 6

### PR DESCRIPTION
**Pull request by ondenman**
Scrapes members of the 6th Convocation.

The last election was on 2016-09-11. We do not yet have the data for this term.

This PR does not scrape contact information. That will be added in a future PR.

* [x] scraper is on Morph.io under the "everypolitician-scrapers" group
* [x] scraper's GitHub "Website" link points at morph.io page
* [x] scraper is set to auto-run
* [x] scraper has webhook set _(usually only set on Wikidata person-data scrapers)_ 
* [x] scraper is configured for archiving _(unless source doesn't require that)_

Scraped page archive
--------------------

* [x] scraper uses _scraped_ gem (unless upstream source has its own archive)
* [x] repo URL uses `https` not `git@` (until [#37 is solved](https://github.com/everypolitician/scraped_page_archive/issues/37)) 
* [x] all gemfile links are secure (e.g., use https)
* [x] morph is configured to write to GitHub ("secret" environment vars)
* [x] pages are archived in new branch of correct scraper repo (yay!)

New data (scraper)
------------------

* [x] scraper gets current (i.e. new term's) members (are they *really* the current members?)
* [x] new term data has correct term set
* [x] fields that are no longer being populated are explained/justified (Contact info is not yet being scraped as its presentation is inconsistent across pages. This will be done in another PR.)
* [x] new fields that are available are being picked up
* [x] names are in same format as previous term
* [x] quantities seem sensible (how many politicians *should* be in this house?)

### CODE REVIEW

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
I'm finding this quite hard to follow, but from what I can see there are (at least) two quite distinct things going on here. I would suggest refactoring to used `Scraped` should happen in a separate PR to the one to add new functionality.

**Comment by ondenman** [comment]
Closing in favour of separate PRs. Starting with: https://github.com/everypolitician-scrapers/belarus-house-of-representatives-2016/pull/4
